setup.cfg

```diff
@@ -53,3 +60,13 @@ commands =
 skipsdist = True
 deps = pylama
 commands = pylama djangosaml2idp
+
+[testenv:typing]
+ignore_outcome = True
+skipsdist = True
+deps =
+    mypy
+    django-stubs
+commands =
+    pip list
```

Comment:
is this a necessary command?

Reply:
Not technically but I do it because it's more clear what the dependencies are vs. how tox normally prints them out.